src/components/page/Loading/index.tsx

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import './style.css';
+
+/** interface for Ripple props */
+export interface RippleProps {
+  borderColor: string;
+  borderStyle: string;
+  borderWidth: string;
+  height: string;
+  minHeight: string;
+  width: string;
+}
+
+export const defaultProps: Partial<RippleProps> = {
+  borderColor: '#ff5d00',
+  borderStyle: 'solid',
+  borderWidth: '4px',
+  height: '64px',
+  minHeight: '80vh',
+  width: '64px',
+};
+
+/** Loading component that displays a nice ripple */
+const Ripple = (props: RippleProps): JSX.Element => {
```

Comment:
I think the more generally accepted way to write this type definition is something like this:

```ts
const Component: React.FC<Props> = (props) => (
```